custom_components/tuya_local/devices/ble_smart_plant_moisture.yaml

```diff
@@ -0,0 +1,42 @@
+name: Moisture sensor
+products:
+  - id: bfbf6c7bb11573ed23j4ya
```

Comment:
Usually BLE devices will have a shorter product id.  Actually the format (valid hex digits except the last 4) looks more like a device id (which is specific to only your device)

Reply:
ooops 😣
gvygg3m8